osquery/core/system.cpp

```diff
@@ -372,8 +372,7 @@ Status createPidFile() {
   }
 
   // If no pidfile exists or the existing pid was stale, write, log, and run.
-  auto pid = boost::lexical_cast<std::string>(
-      PlatformProcess::getCurrentProcess()->pid());
+  auto pid = boost::lexical_cast<std::string>(PlatformProcess::getCurrentPid());
```

Comment:
Nit: We found that std::to_string is faster and easier than a boost lexical cast.